test/github-issues/6195/issue-6195.ts

```diff
@@ -0,0 +1,132 @@
+import "reflect-metadata"
+import { expect } from "chai"
+
+import { DataSource, QueryRunner, Table } from "../../../src"
+import {
+    closeTestingConnections,
+    createTestingConnections,
+    reloadTestingDatabases,
+} from "../../utils/test-utils"
+
+export const testTableName = "test_table"
+export const testColumnName = "test_column"
+export const nonExistentColumnName = "nonexistent_column"
+
+const createTestTable = async (queryRunner: QueryRunner) => {
+    await queryRunner.createTable(
+        new Table({
+            name: testTableName,
+            columns: [
+                {
+                    name: "id",
+                    type: "integer",
+                    isPrimary: true,
+                },
+                {
+                    name: testColumnName,
+                    type: "varchar",
+                },
+            ],
+        }),
+    )
+}
+
+describe("github issues > #6195 feature: fake migrations for existing tables", () => {
+    let dataSources: DataSource[]
+
+    before(async () => {
+        dataSources = await createTestingConnections({
+            entities: [__dirname + "/entity/*{.js,.ts}"],
+            enabledDrivers: [
```

Comment:
why did you explicitly specified all the drivers? it's not scalable, if you want to ignore only one or two drivers - better to add checks in the tests like `if (dataSource.options.type === "mongodb") return`